Weather/Scenes/Weather/WeatherView.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  WeatherView.swift
+//  Weather
+//
+//  Created by Vladislav Kondrashkov on 10/22/18.
+//  Copyright © 2018 Vladislav Kondrashkov. All rights reserved.
+//
+
+import UIKit
+
+final class WeatherView: UIView {
+    var presenter: WeatherPresenter?
+    
+    private let backgroundImage = UIImageView()
+    private let cityLabel = UILabel()
+    private let forecastStatusLabel = UILabel()
+    private let currentTemperatureLabel = UILabel()
+    var timedForecastView = UICollectionView(frame: CGRect.zero, collectionViewLayout: UICollectionViewLayout())
+    private let timedForecastLayout = UICollectionViewFlowLayout()
```

Comment:
Why dont you pass timedForecastLayout to the collection view right away? Make collection view lazy var and it will compile.